README.md

```diff
@@ -41,12 +41,28 @@ Install 10up's command line tool for scaffolding new projects. You can download
 
 `npm run deploy` (build all files for deploy)
 
+`npm run test-a11y` (run accessibility tests)
+
 ## Composer Commands
 
 `composer lint` (lint PHP files)
 
 `composer lint-fix` (lint PHP files and automatically correct coding standard violations)
 
+## Automated Style Guide
+The Theme Scaffolding ships with a default style guide you can find in `/templates/page-styleguide.php`. This file contains all the basic HTML elements you would find at the very top of the cascade (headings, typography, tables, forms, etc.) These base elements will be styled and displayed as you naturally build out your CSS. The style guide also automatically pulls in the color variables used in the project. Any hex codes added into `/assets/css/frontend/global/variables.css` will be displayed in the style guide. To set up your style guide, you just need to create a new page in WordPress and assign it the "Style Guide" template.
```

Comment:
> Any hex codes added…

I understand how this works after looking at the template, but my gut reaction was to wonder if this was accurate. If I add them, they automatically show up in the style guide? Well, yes, because the style guide actually parses the CSS to find all the variables!

Could you make this more explicit about how it works? It could be as simple as

> will **automatically** be displayed in the style

Reply:
@daveross updated